### ISSUE TITLE
bug: Code Snippets are not focusable 

### ISSUE THREAD

**Issue by CBID2** [aidenybai/million#435]
**Describe the bug**
The code examples in the page are hard to read. 
**Possible Solution**
Make them focusable if they have a scroll bar 
**Expected behavior**
The snippets would be easily scrolled by keyboard users.

**Screenshots**

**Device (please complete the following information):**
 - OS: iOS
 - Browser: Chrome
 - Version: 114 

**Additional context**

**Comment by aidenybai** [comment]
I don't think we can fix this since this is handled by Nextra.
